test/unit/org/apache/cassandra/index/sai/cql/StorageAttachedIndexDDLTest.java

```diff
@@ -170,7 +174,7 @@ public void shouldFailCreationUsingMode()
     {
         createTable("CREATE TABLE %s (id text PRIMARY KEY, val text)");
 
-        assertThatThrownBy(() -> executeNet("CREATE CUSTOM INDEX ON %s(val) USING " +
+        assertThatThrownBy(() -> executeNet("CREATE INDEX ON %s(val) USING " +
```

Comment:
so StorageAttachedIndex and sai are both  valid, so the document should be update, I think.

Reply:
Besides I think the purpose of your test should be to test "USING 'sai' " ?

Reply:
"StorageAttachedIndex" support could be removed entirely, but it doesn't hurt to keep it. I'm kind of fine either way. CC @adelapena @mike-tr-adamson

Reply:
The purpose of the test is to make sure we reject invalid `WITH OPTIONS` content, but I'll probably change it to `USING 'sai'`. I just missed it.